FirebaseMLModelDownloader/Sources/CustomModel.swift

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+/// An enum of supported model formats.
+public enum CustomModelFormat {
+  case unknown
+  case tfLite
+  case coreML
+}
+
+/// A custom model that is stored remotely on the server and downloaded to the device.
+public struct CustomModel {
+  public let modelName: String
```

Comment:
nit: If prefix `model` to the property names looks redundant if the intended usage is like the following:
```
let model = CustomModel(...)
print("Name:\(model.modelName)")
```
I would use just `name`, `size`, etc. WDYT?

Reply:
We had an internal discussion about this and the reasoning was that in the future there may be a "name" field that may not refer to the modelName. Just future proofing here. :)

Reply:
Update - turns out we can remove the `model` prefix so I updated the struct.